src/Microsoft.Azure.WebJobs/DefaultResolveValue.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Description
+{
+    /// <summary>
+    /// If set, 
+    /// </summary>
+    public enum DefaultResolveValue
+    {
+        /// <summary>
+        /// Don't default 
+        /// </summary>
+        None,
+
+        /// <summary>
+        /// Default to this function name. 
+        /// This may use the <see cref="FunctionNameAttribute"/> 
```

Comment:
let me try to clarify it

---
In reply to: [113582917](https://github.com/Azure/azure-webjobs-sdk/pull/1124#discussion_r113582917) [](ancestors = 113582917)